package.json

```diff
@@ -10,7 +10,7 @@
     "compile-sass": "sass --quiet-deps --no-source-map --load-path=node_modules/govuk-frontend/dist --load-path=node_modules/@ministryofjustice/frontend --load-path=. assets/scss/application.scss:./assets/stylesheets/application.css assets/scss/print.scss:./assets/stylesheets/print.css --style compressed",
     "watch-ts": "tsc -w",
     "watch-views": "nodemon --watch server/views -e html,njk -x npm run copy-views",
-    "watch-node": "DEBUG=gov-starter-server* nodemon -r dotenv/config --watch dist/ dist/server.js | bunyan -o short",
+    "watch-node": "DEBUG=gov-starter-server* nodemon -r dotenv/config --inspect --watch dist/ dist/server.js | bunyan -o short",
```

Comment:
This looks like it could be useful/interesting .... what does it do?
(I have no problem with it, just interested what it does - I'm guessing something when we run the app locally in dev mode?)

Reply:
It allows you to attach a debugger in intellij. Just add a run config "Attach to nodejs/Chrome" and use the defaults. The annoying thing is the debugger detaches when you edit something and it restarts the process but I found it useful while trying to figure out what was going on in the sorting code...